### PR TITLE
Add doc and classes to improve DX

### PR DESCRIPTION
**Pull request by Nek-**

### CODE REVIEW

**Comment by Nek-** [comment]
Fixes #89

File: README.md [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
 ======
 
-**A websocket library designed to be released.**
+A PHP WebSocket library. With following features:
 
-> This means there is no version  that is going to be release until [v1 milestone](https://github.com/Nekland/Woketo/milestone/1) is complete.
+* Autobahn test suite passed (WebSocket test suite reference)
+* Binary/text messages supported
+* Built on top of reactphp (async socket communication)
+* *Not dependent of any other big framework/library which mean you can use with guzzle (any version) or Symfony (any version)*
```
Comment (valanz) [inline README.md]: * *Not dependent of any other big framework/library which means you can use *it* with guzzle (any version) or Symfony (any version)*

File: README.md [hunk 2]
```diff
@@ -50,57 +43,74 @@ $server->start(); // And that's all <3
...
-```
+```bash
 php tests.php
 wstest -m fuzzingclient
 ```
 
-> wstest is the autobahn test tool. You can install it with `sudo pip install autobahntestsuite`. You can read more on [their documentation](http://autobahn.ws/testsuite/installation.html#installation).
+> wstest is the Autobahn test tool. You can install it with `sudo pip install autobahntestsuite`.
+>
+> You can read more about on [their documentation](http://autobahn.ws/testsuite/installation.html#installation).
```
Comment (valanz) [inline README.md]: You can read more about the installation of Autobahn on ...

File: README.md [hunk 3]
```diff
@@ -50,57 +43,74 @@ $server->start(); // And that's all <3
...
+
+How to something else ?
+-----------------------
+
+* How to learn more about Woketo usage ? [RTFM](docs/index.md) !
+* How to get information about how it work internally ? Read the [docs/dev.md](docs/dev.md) page of doc.
```
Comment (valanz) [inline README.md]: it work*s*

File: README.md [hunk 4]
```diff
@@ -50,57 +43,74 @@ $server->start(); // And that's all <3
...
+* How to contribute ? Read the [CONTRIBUTING.md](CONTRIBUTING.md) page of doc.
+* How to get support ? [Use Gitter](http://gitter.im/Nekland/Woketo), _the issue tracker is **not** a forum_.
+
+What's next ?
+-------------
+
+You can see what's plan for next versions in the [github milestones](https://github.com/Nekland/Woketo/milestones).
```
Comment (valanz) [inline README.md]: what's *planned*

File: README.md [hunk 5]
```diff
@@ -50,57 +43,74 @@ $server->start(); // And that's all <3
...
+
+What Woketo does _not_ do ?
+---------------------------
+
+Currently there is no support of the following:
+
+- WebSocket client (you still need to use [pawl](https://github.com/ratchetphp/Pawl), but we're working on)
```
Comment (valanz) [inline README.md]: we're working on it

**Pull request merged**